apps/coscrad-frontend-e2e/src/e2e/tags/create-tag.e2e.cy.ts

```diff
@@ -21,14 +21,82 @@ describe(`Create Tag`, () => {
         });
 
         describe(`when the form is complete`, () => {
```

Comment:
Add cases for
- when the form is not complete
- when attempting to create a tag with a duplicate label (should fail)